book/src/generated/lang-support.md

```diff
@@ -93,6 +93,8 @@
 | pascal | ✓ | ✓ |  | `pasls` |
 | perl | ✓ | ✓ | ✓ |  |
 | php | ✓ | ✓ | ✓ | `intelephense` |
+| ponylang | ✓ | ✓ | ✓ |  |
+| ponylang | ✓ | ✓ | ✓ |  |
```

Comment:
This seems duplicated
```suggestion
```

Reply:
I added the ponylang entry two times to the languages.toml, oopsie!